content/includes/footer.html

```diff
@@ -96,11 +96,22 @@ <h4 class="va-crisis-line">Get help from Veterans Crisis Line</h4>
 <script src="/js/user-voice.js"></script>
 
 <!-- htmllint attr-bans="[]" -->
-<!-- Google Tag Manager -->
-<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-WFJWBD"
-height="0" width="0" class="csp-inline-patch-footer"></iframe></noscript>
-<script src="/js/google-analytics.js"></script>
-<!-- End Google Tag Manager -->
+{% if buildtype === 'development' %}
+  <script>
+    window.dataLayer = [];
```

Comment:
In development, this just mocks out the dataLayer so that later calls to `window.dataLayer.push()` don't raise JS errors. We're turning off GA in development, so there's no need to send real data.